package.json

```diff
@@ -28,10 +30,21 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src || printf '\\n\\e[31mRun lint-fix to fix\\e[0m\\n\\n'",
+    "lint-fix": "eslint --fix src",
+    "prettier": "prettier --check '**/*.{css,scss,js}' || printf '\\n\\e[31mRun prettier-fix to fix\\e[0m\\n\\n'",
+    "prettier-fix": "prettier --write '**/*.{css,scss,js}'"
   },
-  "eslintConfig": {
-    "extends": "react-app"
+  "lint-staged": {
+    "src/**/*.{css,scss,js}": [
+      "eslint --fix src"
```

Comment:
Thanks! Can you also include the `prettier --write` command?

Reply:
It's not super obvious but I've configured eslint with a plugin[1] which will run `prettier --write` before linting, and running with `--fix` will run it in write mode.

[1] https://github.com/covid-projections/covid-projections/pull/173/files#diff-e4403a877d80de653400d88d85e4801aR7

Reply:
oh cool! thanks for adding and pointing that out.